### ISSUE TITLE
How long does it take to process a frame of video?

### ISSUE THREAD

**Issue by mayidu** [qqwweee/keras-yolo3#253]
hello, Can you tell me how long does it takes to process a frame of video?
I test it need 500-600ms， Feeling a bit slow！

**Comment by Psyf** [comment]
Sounds like what I'm getting by using webcam directly too

**Comment by mayidu** [comment]
yes! Do you have test time, how long it takes to process a frame?

**Comment by Psyf** [comment]
I'm getting 350ms on average. I wonder what the bottleneck is here.

**Comment by mayidu** [comment]
I don't know! And I test yolov2 in tensorflow, also very slow!

**Comment by ghost** [comment]
If you tested in large frame, try size it smaller and resize the output bounding box as original video size

**Comment by mayidu** [comment]
NO！ I test the frame is small and the network size is 128*128, but also slow

**Comment by YunYang1994** [comment]
https://github.com/YunYang1994/tensorflow-yolov3

**Comment by u53216** [comment]
Have you konwn why the test time is about 35ms，much slower than the Darknet？I have met the problem，too。
